From 73c690ca9bd1d470c603f5e1ee48d2384941ae55 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Tue, 19 Feb 2019 13:55:12 -0500
Subject: [PATCH] Fix integer sizes used with ap_set_flag_slot()

ap_set_flag_slot() requires a field of type `int`.  Previously we
passed type `bool` in two places, causing test failures on s390x
because logging was not correctly configured.

Signed-off-by: Robbie Harwood <rharwood@redhat.com>
(cherry picked from commit f89b876be5619d750e6ab4ea6e730a9a26b1cac4)
---
 src/mod_auth_gssapi.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mod_auth_gssapi.h b/src/mod_auth_gssapi.h
index 71404ff..8c0b972 100644
--- a/src/mod_auth_gssapi.h
+++ b/src/mod_auth_gssapi.h
@@ -79,7 +79,7 @@ struct mag_config {
     gid_t deleg_ccache_gid;
     gss_key_value_set_desc *cred_store;
     bool deleg_ccache_unique;
-    bool s4u2self;
+    int s4u2self;
     char *ccname_envvar;
 #endif
     struct seal_key *mag_skey;
@@ -90,7 +90,7 @@ struct mag_config {
     bool negotiate_once;
     struct mag_name_attributes *name_attributes;
     const char *required_na_expr;
-    bool enverrs;
+    int enverrs;
     gss_name_t acceptor_name;
     bool acceptor_name_from_req;
 };
